gmm/handler.go

```diff
@@ -1294,7 +1294,7 @@ func handleRequestedNssai(ue *context.AmfUe, anType models.AccessType) error {
 					UserLocation:     &ue.Location,
 					RrcEstCause:      ue.RanUe[anType].RRCEstablishmentCause,
 					UeContextRequest: ue.RanUe[anType].UeContextRequest,
-					AnN2IPv4Addr:     ue.RanUe[anType].Ran.Conn.RemoteAddr().String(),
+					AnN2IPv4Addr:     ue.RanUe[anType].Ran.RemoteAddr,
```

Comment:
Need to make sure Ran Remote address is correct.

Reply:
we should get remote address and ran id separately. Looks like we need both fields.

Reply:
Change remoteaddr variable name